benchmarks/lasso/solvers/cd.py

```diff
@@ -30,7 +30,7 @@ class Solver(BaseSolver):
     name = "cd"
 
     install_cmd = 'pip'
-    requirements = ['numba']
+    requirements = ['numba', 'sparse']
```

Comment:
I don't think that would work, `sparse` is not the package name, it is the module name.
```suggestion
    requirements = ['numba', 'scipy']
```

Reply:
Looks way better thanks @TomDLT  and @agramfort